en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg

```diff
@@ -147,12 +147,12 @@
          style="font-style:italic;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:35.77793884px;line-height:1.25;font-family:Lavi;-inkscape-font-specification:'Lavi Italic'"
          x="1856.9244"
          y="283.55594"
-         id="tspan470">public holidays. Offer is</tspan><tspan
+         id="tspan470">public holidays. Offer may</tspan><tspan
          sodipodi:role="line"
          style="font-style:italic;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:35.77793884px;line-height:1.25;font-family:Lavi;-inkscape-font-specification:'Lavi Italic'"
          x="1856.9244"
          y="328.27838"
-         id="tspan472">not cumulative. Limited to</tspan><tspan
```

Comment:
Nº of results in a search engine, as an indication of usage:

* `offer "non cumulative"`: 371 000
* `offer "may not be combined"`: 46 000 000